pkg/gadgets/run/tracer/run.go

```diff
@@ -77,13 +80,92 @@ func (g *GadgetDesc) ParamDescs() params.ParamDescs {
 			Description: "Gadget definition in yaml format",
 			TypeHint:    params.TypeBytes,
 		},
+		// Hardcoded for now
+		{
+			Key:          "authfile",
+			Title:        "Auth file",
+			DefaultValue: oci_helper.DefaultAuthFile,
```

Comment:
I think this should be briefly documented in docs/image.md, even if it is at first just listing the commands to run:

```
sudo -E ig image build...
sudo -E ig image list
sudo -E ig image pull...
sudo -E ig image push...
```

Btw, the `image push` command works fine for me even without anything in `/var/lib/ig/config.json`. Does it automatically picks up `~/.docker/config.json`? If so, I think that should be documented. How does it know the home directory? Is it because `sudo -E` maintains the `$HOME` environment variable?

Reply:
> Does it automatically picks up ~/.docker/config.json

Yes, there's a fallback mechanism to use docker credentials: https://github.com/inspektor-gadget/inspektor-gadget/blob/6be6968dce17113a7eb62ff7a834649bd8d0cdf8/pkg/oci_helper/oci_helper.go#L257

> I think that should be documented

Will push some documentation in the next iteration.

> Is it because sudo -E maintains the $HOME environment variable

I believe so.